app/views/blog/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> Create a new blog </h1>
+<%= form_with model: @blog, local: true do |form| %>
```

Comment:
This is calling on an instance variable in the new method. add the instance variable @blog = Blog.new